android/quest/src/test/java/org/smartregister/fhircore/quest/app/fakes/FakeFhirEngineModule.kt

```diff
@@ -26,11 +26,9 @@ import dagger.hilt.components.SingletonComponent
 import dagger.hilt.testing.TestInstallIn
 import io.mockk.spyk
 import javax.inject.Singleton
-import org.smartregister.fhircore.engine.di.DispatcherModule
 import org.smartregister.fhircore.engine.di.FhirEngineModule
-import org.smartregister.fhircore.engine.di.NetworkModule
 
-@Module(includes = [NetworkModule::class, DispatcherModule::class])
```

Comment:
Are the dependencies from these modules no longer required when instantiating dependencies in `FakeFhirEngineModule`?

Reply:
The includes may have been unnecessary